assets/app/view/game/par_chart.rb

```diff
@@ -15,7 +15,7 @@ class ParChart < Snabberb::Component
 
       def render
         @step = @game.round.active_step
-        @current_entity = @step.current_entity
+        @current_entity = @step&.current_entity
         @current_operator = @game.respond_to?(:current_operator) ? @game.current_operator : nil
```

Comment:
at end game the market view fails as there is no @step current entity. This is used for parring logic only